pkg/asset/installconfig/openstack/validation/machinepool.go

```diff
@@ -119,8 +126,8 @@ func validateFlavor(flavor Flavor, req flavorRequirements, fldPath *field.Path)
 	if flavor.VCPUs < req.VCPUs {
 		errs = append(errs, fmt.Sprintf("Must have minimum of %d VCPUs, had %d", req.VCPUs, flavor.VCPUs))
 	}
-	if flavor.Disk < req.Disk {
-		errs = append(errs, fmt.Sprintf("Must have minimum of %d GB Disk, had %d GB", req.Disk, flavor.Disk))
+	if flavor.Disk < req.Storage && flavor.Ephemeral < req.Storage {
```

Comment:
I need to change the condition here as well, I'll fix it today. Please don't review that line yet.